composeApp/src/commonMain/kotlin/org/paystell/app/screens/authentication/SignUpScreen.kt

```
@@ -0,0 +1,368 @@
+                        delay(1500)
+                        
+                        // Simulate random network error (20% chance)
+                        if (Random.nextDouble() < 0.2) {
+                            throw Exception("Network error")
+                        }
+                        
+                        // Success path
+                        showSuccessDialog = true
+                        delay(1500) // Show success message briefly
+                        onSignUpSuccess()
+                    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)